cmd/create/cluster/cmd.go

```diff
@@ -1150,20 +1149,20 @@ func run(cmd *cobra.Command, _ []string) {
 					billingAccount = aws.ParseOption(billingAccount)
 				}
 
-				err := validateBillingAccount(billingAccount)
+				err := ocm.ValidateBillingAccount(billingAccount)
 				if err != nil {
 					r.Reporter.Errorf("%v", err)
 					os.Exit(1)
 				}
 
 				// Get contract info
-				contracts, isContractEnabled := GetBillingAccountContracts(cloudAccounts, billingAccount)
+				contracts, isContractEnabled := ocm.GetBillingAccountContracts(cloudAccounts, billingAccount)
 
 				if billingAccount != awsCreator.AccountID {
 					r.Reporter.Infof(
-						"The selected AWS billing account is a different account than your AWS infrastructure account." +
+						"The selected AWS billing account is a different account than your AWS infrastructure account. " +
```

Comment:
suggested rewrite: 

The AWS billing account you selected is different from your AWS infrastructure account.

OR

Select your AWS infrastructure billing account. The AWS account you selected is not your AWS infrastructure account. 

OR

Select your AWS infrastructure billing account. The AWS account you selected is not the correct account.

Reply:
+1 "The AWS billing account you selected is different from your AWS infrastructure account."

Reply:
updated as suggested